SWLOR.Game.Server/Feature/PerkDefinition/RangedPerkDefinition.cs

```diff
@@ -658,5 +659,18 @@ private void CripplingShot()
                 .RequirementCharacterType(CharacterType.Standard)
                 .GrantsFeat(FeatType.CripplingShot3);
         }
+
+        private void ZenMarksmanship()
+        {
+            _builder.Create(PerkCategoryType.RangedGeneral, PerkType.ZenMarksmanship)
+                .Name("Zen Marksmanship")
+
+                .AddPerkLevel()
+                .Description("Your ranged attacks now use Willpower for damage if your Willpower is higher than your Perception, for Pistols and Rifles, or your Might, for Shurikens.")
```

Comment:
This is worded too specifically. Change it to something more akin to

"Your ranged attacks now use Willpower for damage if your Willpower is higher than your weapon's damage stat."

Reply:
Amended.